src/App.tsx

```diff
@@ -15,58 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState<string>('Jam');
```

Comment:
<string> is unnecessary because TypeScript can automatically infer type from initial value ;)